librz/core/cconfig.c

```diff
@@ -3194,7 +3194,7 @@ RZ_API int rz_core_config_init(RzCore *core) {
 	SETICB("bin.maxstrbuf", 1024 * 1024 * 10, &cb_binmaxstrbuf, "Maximum size of range to load strings from");
 	n = NODECB("bin.str.enc", "guess", &cb_binstrenc);
 	SETDESC(n, "Default string encoding of binary");
-	SETOPTIONS(n, "ascii", "8bit", "utf8", "utf16le", "utf32le", "utf16be", "utf32be", "guess", NULL);
+	SETOPTIONS(n, "ascii", "utf8", "utf16le", "utf32le", "utf16be", "utf32be", "ibm037", "ibm290", "ebcdices", "ebcdicuk", "ebcdicus", "guess", NULL);
```

Comment:
I don't like this particular change - there are many 8bit encodings and it's better to keep this option as is.